src/index.ts

```diff
@@ -122,13 +130,15 @@ export type FileSystemShortHands = {
  * - `authority`, the authority system, request or provide authority to parts of (or the entire) file system and account system.
  * - `fileSystem`, the file system.
  *
- * This object also has a few other functions, for example to load a filesystem.
- * These are called "shorthands" because they're the same functions available
- * through other places in the ODD SDK, but you don't have to pass in the components.
+ * An ODD program revolves around two main things, the file system (WNFS) and authorization (UCANs).
+ * Both of these can be used entirely locally without depending on an external service.
  *
- * See `assemble` for more information. Note that this function checks for browser support,
- * while `assemble` does not. Use the latter in case you want to bypass the indexedDB check,
- * which might not be needed, or available, in certain environments or using certain components.
+ * On top of that we've built components that can be swapped out with different
```

Comment:
```suggestion
 * We've built components that can be swapped out with different
```

This reads alright without the transition phrase.